### ISSUE TITLE
a small question from a green hand

### ISSUE THREAD

**Issue by thereisagirliloveinHZ** [intel-aero/meta-intel-aero#413]
If i want to develop the obstacle avoidance function of drones, can i only apply to the yocto project without installing an ubuntu system?

**Comment by zehortigoza** [comment]
Yes, it is just a matter of what distro is easier for you to build and load your application.
With Ubuntu you have "apt get" to install libraries with Yocto you would have to build your application with static libraries or build a Yocto image with the libraries you need.

**Comment by githublearner-li** [comment]
thank you very much for your response! i want to know whether Using Ubuntu to develope applications is easier than building applications with yocto ,thank you!
